src/containers/Transactions/DetailTab.jsx

```diff
@@ -20,21 +21,21 @@ class DetailTab extends Component {
     const { t, language, data } = this.props
     const { TransactionResult } = data.meta
     const time = localizeDate(new Date(data.date), language, DATE_OPTIONS)
+    const code = '' // Never directly used but <Trans> needs it
     let line1
 
     if (TransactionResult === SUCCESSFULL_TRANSACTION) {
       line1 = t('successful_transaction')
     } else {
       line1 = (
-        <>
-          {t('fail_transaction')}
-          <span className="tx-result fail">{TransactionResult}</span>
-        </>
+        <Trans i18nKey="fail_transaction" values={{ code: TransactionResult }}>
+          <span className="tx-result fail">{{ code }}</span>
```

Comment:
Why create a separate variable instead of doing this?
```suggestion
          <span className="tx-result fail">{{ '' }}</span>
```

Reply:
I did not see that as an option in the react-i18n. I will try that out

Reply:
Oh, I see what's happening - you're passing in an object. This is confusing to read, I suggested doing this instead:
```suggestion
          <span className="tx-result fail">{ { code: '' } }</span>
```
(and same spacing for the other places where you place in a constructed object

Reply:
I am actually not placing in an object.  It is doing interpolation when i18n-next evaluates it.  Type checking gets in the way and thinks it is ts code.

Reply:
> Why create a separate variable instead of doing this?

`<span className="tx-result fail">{{ '' }}</span>` turns into `<span className="tx-result fail"/>` due to linter but works.

`<span className="tx-result fail">{{ code: '' }}</span>` also works but I found it more confusing.  I added an assertion to make sure the span gets populated with the code properly.